src/utils/shared.ts

```diff
@@ -149,6 +152,42 @@ export const clearEmptyObjectValues = (obj: any): any => {
   return answer;
 };
 
+type ErrorHandlerObj = {
+  errorHandler: string;
+  description: string;
+};
+/**
+ * This method returns the error handler object based on the error handler strategy string
+ * @param errorHandler
+ * @returns ErrorHandlerObj
+ */
+export const returnErrorHandlersNames = (
```

Comment:
I feel like this function would probably live better alongside the component either in `ErrorHandler.tsx` or alongside it, closer to it's usage.  Maybe the type too.  Only because it's specific to the connectors concept of configuring the error handling, and this function mostly seems to deal with presentation of the data.

Reply:
updated